doc/source/user_guide/migration_guide.ipynb

```diff
@@ -0,0 +1,7981 @@
+{
```

Comment:
<p>plot_hdi is missing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/2459/discussion/#comment-2170791353'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>